sdk/storage/azblob/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## v0.1.0 (Unreleased)
+* This is the initial preview release of the `azblob` library
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
```

Comment:
The bullet point should be under "Features Added", then remove the other headers